metacontrol_sim.rosinstall

```diff
@@ -0,0 +1,12 @@
+- git:
+    local-name: metacontrol_sim
+    uri: https://github.com/rosin-project/metacontrol_sim
+- git:
+    local-name: yumi
+    uri: https://github.com/marioney/yumi
+- git:
+    local-name: metacontrol_nav
+    uri: https://github.com/marioney/metacontrol_nav
```

Comment:
The metacontrol_nav package has been moved to the rosin-project.  The correct address here should be

https://github.com/rosin-project/metacontrol_nav

Reply:
Change done and history rebased! Now should be fine :+1: